### DISCUSSION TITLE
How can I import files as text

### DISCUSSION THREAD

**Discussion by LokiMidgard** [sveltejs/kit#4857] [Q&A] [answered]
I need to import a xml file.

I used `import nota from './../data/nota.xml';` in the script block of my svelte file and made following declaration in a `typeings.d.ts` file:
```ts
declare module '*.xml' {
    const content: string;
    export default content;
  }
```

When running the App I get following error. 

```
Failed to parse source for import analysis because the content contains invalid JS syntax. You may need to install appropriate plugins to handle the .xml file format.
```

I assume the file gets loaded as JS. How can I tell svelte to just load the file as text?

**Comment by bluwy** [comment, marked as answer]
Use the `?raw` suffix: `import nota from './../data/nota.xml?raw';` https://vitejs.dev/guide/assets.html#importing-asset-as-string
